spec/lib/scrabble_spec.rb

```diff
@@ -0,0 +1,50 @@
+require './lib/scrabble/scrabble.rb'
+
+describe Scrabble::Scrabble do
```

Comment:
I like the structure of the test.  Clear and easy to read what you are testing.  As another developer I love that.  What can be improved is testing the highest_score_from method.  I would like to see tests for sets of words that meet certain boundary cases (not equal in point value, equal in point but not in length, equal in point and length).